src/cljs/main/broadfcui/page/billing/billing_management.cljs

```diff
@@ -151,24 +151,37 @@
   {:render
    (fn [{:keys [props]}]
      (let [{:keys [project-name]} props]
-       [:div {:style {:padding "1em"}}
+       [:div {:style {:padding "1rem 1rem 0"}}
         [:div {:style {:marginBottom "1rem" :fontSize "1.1rem"}}
-         [comps/Breadcrumbs {:crumbs [{:text "Billing Management" :href (nav/get-link :billing)}
-                                      (when project-name {:text project-name})]}]]
+         [:div {:style {:marginBottom "1rem" :fontSize "1.1rem"}}
+          [:div {:style {:fontSize "1.2em"}} (when project-name "Billing Project: ")
+           [:span {:style {:fontWeight 500}} (if project-name project-name "Billing Management")]]]]
         (if project-name
-          [BillingProjectManagementPage (utils/restructure project-name)]
+          [MembershipManagementPage
+           {:group-name project-name
+            :add-endpoint #(endpoints/add-billing-project-user {:project-id %1
+                                                                :role %2
+                                                                :user-email %3})
```

Comment:
If these functions were called with the map already constructed, instead of 3 positional arguments, this would then boil down to `:add-endpoint endpoints/add-billing-project-user`